<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="5" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0" rowspan="4">
         <widget class="QLabel" name="picLabel">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Leave a suggestion:</string>
          </property>
          <property name="buddy">
           <cstring>lineEdit_2</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="enterPushButton">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700;&quot;&gt;Welcome to&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt; font-weight:700; font-style:italic; text-decoration: underline; color:#ff0000;&quot;&gt;AIR BIKE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Enjoy!</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="submitButton">
          <property name="text">
           <string>Submit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="10" alignment="Qt::AlignRight">
         <widget class="QPushButton" name="pushButton">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="17" column="7" colspan="3" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="infoLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="7">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Welcome!</string>
          </property>
         </widget>
        </item>
        <item row="6" column="7" colspan="3" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Need an account? </string>
          </property>
         </widget>
        </item>
        <item row="3" column="7" colspan="3" alignment="Qt::AlignHCenter">
         <widget class="QLineEdit" name="emailLogin">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>email</string>
          </property>
         </widget>
        </item>
        <item row="7" column="7" colspan="3" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="signInButton">
          <property name="maximumSize">
           <size>
            <width>100000</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string>Sign in</string>
          </property>
         </widget>
        </item>
        <item row="5" column="7" colspan="3" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="submitPushButton">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item row="2" column="7" colspan="3" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Log In</string>
          </property>
         </widget>
        </item>
        <item row="4" column="7" colspan="3" alignment="Qt::AlignHCenter">
         <widget class="QLineEdit" name="passwordLogin">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>password</string>
          </property>
         </widget>
        </item>
        <item row="1" column="7" colspan="3">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="8">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="7">
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="8" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="wrongPass">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="13" column="0">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="8" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Email:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLineEdit" name="username"/>
        </item>
        <item row="9" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLineEdit" name="password1">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="15" column="0" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Create an account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Password (again):</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLineEdit" name="password2">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="14" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Already have an account?</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" alignment="Qt::AlignHCenter">
         <widget class="QLineEdit" name="email"/>
        </item>
        <item row="12" column="0" alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="registerButton">
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="2">
         <widget class="QPushButton" name="realArticlesButton">
          <property name="text">
           <string>ARTICLES</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Check your articles:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2" alignment="Qt::AlignRight|Qt::AlignTop">
         <widget class="QPushButton" name="backButton4">
          <property name="text">
           <string>Log out!</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="warning2Bikes">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Check history:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="picLabel2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="1" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="welcomeLine">
          <property name="text">
           <string>Welcome</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="artikalSetButton">
          <property name="text">
           <string>SET</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Set an article on the market</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="atrikliButton">
          <property name="text">
           <string>HISTORY</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="1" alignment="Qt::AlignRight">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#ff007f;&quot;&gt;We wish you a great success!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="bikeGearLine"/>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="bikeSellLine"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="backArtikli">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Bike gear:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>For how much did you buy it:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>How much are you selling it for:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="bikeNameLine"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Bike name:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="bikeBoughtLine"/>
        </item>
        <item row="6" column="1" alignment="Qt::AlignRight">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>FINISH</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="0">
         <widget class="QTableView" name="tabela1"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>TRANSACTION HISTORY</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="backTable">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_7">
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>112</x>
          <y>440</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>MY ARTICLES</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableViewArticles">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>191</y>
          <width>571</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>70</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Go Back</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
